routes/camera.js

```diff
@@ -0,0 +1,117 @@
+import express from "express";
+const router = new express.Router();
+import http from "http";
+
+router.get("/save/:id(\\d\\d?)/", (req, res) => {
+    let id = req.params.id;
+    if (id.length === 1) {
+        id = `0${id}`;
+    }
+    http.get({
+        host: "192.168.10.101",
```

Comment:
The cameras will be coupled next week. The cameras will have an id assigned and some other properties.